dist/vespa.spec

```diff
@@ -466,6 +466,15 @@ VERSION=%{version} CI=true make -C client/go install-all
 
 %check
 %if ! 0%{?installdir:1}
+%if 0%{?_java_home:1}
+export JAVA_HOME=%{?_java_home}
+%else
+export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+%endif
+export PATH="$JAVA_HOME/bin:$PATH"
+%if 0%{?el8}
+python3.9 -m pip install --user pytest
```

Comment:
Not a good idea to install stuff as part of the rpmbuild ?

Reply:
Not installing pytest causes a failure due to pytest not being installed.

Reply:
Ok. I do wish we could express this dependency with `BuildRequires`.

Reply:
Is this due to ANN testing ?

Reply:
It is for the ANN benchmark unit test.